.github/workflows/deploy.yml

```diff
@@ -14,10 +14,10 @@ jobs:
         with:
           token: ${{ secrets.LOBOT_TOKEN }}
           ref: "main"
-      - name: Use Node.js 14.x
+      - name: Use .node-version Node.js version
         uses: actions/setup-node@v4
         with:
-          node-version: 14.x
+          node-version-file: ".node-version"
```

Comment:
Simplifying the CI workflow a bit by sharing the same node version as we use for `nodenv`.